src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -58,5 +65,20 @@ object Boot extends App with LazyLogging {
       }
   }
 
+  private def startClusterMonitors(dbRef: DbReference, clusterMonitor: ActorRef)(implicit executionContext: ExecutionContext) = {
+    dbRef.inTransaction { dataAccess =>
+      dataAccess.clusterQuery.listMonitored.map { clusters =>
+        clusters.foreach { cluster =>
+          cluster.status match {
```

Comment:
I can throw in a couple monoids if you like :)

Actually it would probably be good to move the looping/actor notification out of the transaction. Perhaps:
```
  private def startClusterMonitors(dbRef: DbReference, clusterMonitor: ActorRef)(implicit executionContext: ExecutionContext) = {
    dbRef.inTransaction { dataAccess =>
      dataAccess.clusterQuery.listMonitored
    } onComplete {
      case Success(clusters) =>
        clusters.foreach {
          case c if c.status == ClusterStatus.Deleting => clusterMonitor ! ClusterDeleted(c)
          case c => clusterMonitor ! ClusterCreated(c)
        }
      case Failure(e) =>
        logger.error("Error starting cluster monitor", e)
    }
  }
```